### ISSUE TITLE
Invoke-PnPTenantTemplate : Unknown Error

### ISSUE THREAD

**Issue by manuhinCCS** [pnp/sp-starter-kit#690]
## Category

- [x] Bug

## Describe the bug

Getting below error when trying to run the command.

Command: Invoke-PnPTenantTemplate -Path starterkit.pnp -Parameters @{"PORTALURL"="/sites/kitv3portal"; "MARKETINGALIAS"="kitv3marketing";   "HRALIAS"="kitv3hr" }

Invoke-PnPTenantTemplate : Unknown Error
At line:1 char:1
+ Invoke-PnPTenantTemplate -Path starterkit.pnp -Parameters @{"PORTALUR ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Invoke-PnPTenantTemplate] 
   , PSInvalidOperationException
    + FullyQualifiedErrorId : InvalidOperation,PnP.PowerShell.Commands.Provisi 
   oning.Tenant.InvokeTenantTemplate

#### Observed Behavior

![image](https://github.com/pnp/sp-starter-kit/assets/112081775/3af05e8b-ed91-4dca-9b1a-af1ed6c479f1)

## Environment Details (*Development & Target environment*)

- **Starter kit version**: v3
- **OS**: Windows 10
- **Target Environment**: SharePoint Online
- **Framework**: Node.js v14.20.0
- **PnPPowerShell Version** : 1.12.0
- **Browser(s)**:  Chrome

Thanks for your contribution! Sharing is caring.

**Comment by eoverfield** [comment]
I don't think this is the issue, though try adding .\ before starterkit.pnp, i.e. 
`Invoke-PnPTenantTemplate -Path .\starterkit.pnp`

**Comment by manuhinCCS** [comment]
> I don't think this is the issue, though try adding .\ before starterkit.pnp, i.e. `Invoke-PnPTenantTemplate -Path .\starterkit.pnp`

Still, not working.

**Comment by gitr4w** [comment]
Same issue, any solution?
